Library/Homebrew/cmd/update.sh

```diff
@@ -249,6 +252,8 @@ pull() {
     pop_stash_message
   fi
 
+  [[ -n "$HOMEBREW_VERBOSE" ]] && echo
```

Comment:
Fetch doesn't echo every time; put it at the beginning of this line and you will sometimes see the first line of output as an empty line.